ProductsWPF/EditProduct.xaml.cs

```diff
@@ -0,0 +1,61 @@
+﻿using AccessServices.DTOs;
+using AccessServices.Infrastructure;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Documents;
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using System.Windows.Shapes;
+
+namespace ProductsWPF
+{
+    /// <summary>
+    /// Interaction logic for EditProduct.xaml
+    /// </summary>
+    public partial class EditProduct : Window
+    {
+        ProductService productService;
+        ProductDTO _product;
+        public event Action ItemEdited;
+        IEnumerable<CategoryDTO> categories;
+        CategoryDTO _category { get { return categories.Single(c => _product.CategoryName == c.Name); } set { } }
```

Comment:
You can remove set or use just
CategoryDto Category => categories.Single(c => _product.CategoryName == c.Name);